src/main/resources/com/google/api/codegen/php/test.snip

```diff
@@ -102,27 +127,39 @@
         @end
     @end
     
-    @switch test.clientMethodType
-    @case "OptionalArrayMethod"
-        {@optionalArraySuccessAsserts(test)}
-    @case "PagedOptionalArrayMethod"
-        {@pagedOptionalArraySuccessAsserts(test)}
-    @default
-        $unhandled case: {@test.clientMethodType.toString}$
-    @end
 @end
 
-@private optionalArraySuccessAsserts(test)
+@private successAsserts(test)
+    @if test.hasReturnValue
+        $response = $client->{@test.clientMethodName}({@sampleMethodCallArgList(test.initCode.fieldSettings)});
+        
+    @else
+        $client->{@test.clientMethodName}({@sampleMethodCallArgList(test.initCode.fieldSettings)});
+        
+    @end
+    {@singleCallSuccessAsserts(test)}
```

Comment:
It might be more readable to have fewer ifs but slightly more repetition:

```
@if test.hasReturnValue
    $response = ...
    {@singleCallSuccessAsserts(test)}
    $this->assert...
@else
    $client->...
    {@singleCallSuccessAsserts(test)}
@end
```

Or is it necessary to have the `$expectedResponse` assertion after `singleCallSuccessAsserts`? If not:


```
@if test.hasReturnValue
    $response = ...
    $this->assert...
@else
    $client->...
@end

{@singleCallSuccessAsserts(test)}
```